Add information about supported PMM versions (as 3 is not supported) (#215)
percona/everest-doc: docs/use/monitor_endpoints.md

Percona Everest provides monitoring capabilities with PMM to maintain a reliable and secure database infrastructure.

!!! caution alert alert-warning "Important"

>     Currently, only PMM v2.x is supported. The support for PMM v3.x is going to be added in the future releases

Reviewer: ```suggestion
    PMM v2.x is currently supported. Support for PMM v3.x is planned for future releases.
```
